### PR TITLE
fix: Increase counter after an execution

### PR DESCRIPTION
**Pull request by jack4it**
## Pull request description 

Test execution counter is not increased after execution finishes, i.e. the result is stuck at `running` even after the execution succeeded or failed. This change fixes that.

```
testkube_executions_count{name="xyz",result="running",type="universal-package-test"} 1
```

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]
LGTM

File: pkg/jobs/jobclient.go
Comment (vsukhin) [inline pkg/jobs/jobclient.go:279]: Actually, worth to add it to LaunchK8sJobSync method as well. Methods need to be merged, treat it as a technical debt for now

Comment (jack4it) [reply pkg/jobs/jobclient.go:282]: Sure, done

**Comment by jack4it** [comment]
hey @vsukhin apparently this PR is blocked by the rule `First-time contributors need a maintainer to approve running workflows`. do you happen to know who the maintainer is for this repo? looking forward to merging and testing it out. thx!

**Comment by exu** [comment]
@jack4it @vsukhin sorry for this lag - just catching up everything after Kubeshop meeting in Stockholm ;)

**Comment by vsukhin** [comment]
@jack4it sounds like "import cycle not allowed" compiler error

**Comment by jack4it** [comment]
> @jack4it sounds like "import cycle not allowed" compiler error

can you rerun the workflow? thx!

**Pull request merged**
